pkg/gopass/secret/ident.go

```diff
@@ -0,0 +1,6 @@
+package secret
+
+const (
+	// Ident is the Gopass MIME secret header
+	Ident = "GOPASS-SECRET-1.0"
```

Comment:
we might want to mention it's deprecated? Also why not in parsesec with the Legacy function?

Reply:
The secrets implementation needs to change once we figure out what to expose to the Gopass API users. I plan to clean up the remaining pieces by then.